ingestion/functions/parsing/colombia/colombia.py

```diff
@@ -0,0 +1,223 @@
+            "Colombia"]
+        if term != '']
+
+    return {"query": ", ".join(query_terms)}
+
+
+def make_notes(raw_entry):
+
+    return notes
+
+
+def get_ethnicity(raw_entry):
+
+    ethnicity_map = {'Otro': 'Other',
+                     'Negro': 'Black',
+                     'Indigeno': 'Indigenous'}
+
+    if ethnicity_map.get(raw_entry['Pertenencia etnica'], None):
+        if raw_entry['Pertenencia etnica'] == 'Indígena':
+            if raw_entry['Nombre grupo etnico'] != '' and raw_entry['Nombre grupo etnico'] != 'Sin Comunidad' and raw_entry['Nombre grupo etnico'] != 'Sin definir':
+                ethnicity = f"Indigenous, {raw_entry['Nombre grupo etnico']}"
+            else:
+                ethnicity = "Indigenous"
+        else:
+            ethnicity = ethnicity_map.get(
+                raw_entry['Pertenencia etnica'], None)
+
+    else:
+        ethnicity = "Unknown"
+
+    return ethnicity
+
+
+def parse_cases(raw_data_file, source_id, source_url):
+    """
+    Parses G.h-format case data from raw API data.
+
+    Caveats:
+    - Assuming the date confirmed is the date of diagnosis (Fecha diagnostico) rather than
+    Fecha de notificación (generally several days earlier). When date of diagnosis, using date reported online as proxy.
+
+    - Tipo recuperación refers to how they decided the patient had recovered: either by 21 days elapsing since
+    symptoms, or a negative PCR/antigen test
+
+    - No dates for travel history, only distinction is between cases of type: 'Importado' vs. 'Relacionado'
+    """
+
+    symptom_map = {'Leve': 'Mild',
+                   'Moderado': 'Moderate',
+                   'Grave': 'Serious'}
+
+    with open(raw_data_file, "r") as f:
+        reader = csv.DictReader(f)
+        for entry in reader:
+            case = {
+                "caseReference": {
+                    "sourceId": source_id,
+                    "sourceEntryId": entry["ID de caso"],
+                    "sourceUrl": source_url
+                },
+                "location": convert_location(entry),
+                "notes": '',
+                "demographics": {
+                    "ageRange": {
+                        "start": float(entry["Edad"]),
+                        "end": float(entry["Edad"])
+                    },
+                    "gender": convert_gender(entry["Sexo"]),
+                    "ethnicity": get_ethnicity(entry)
+                }
+
+            }
+
+            if entry["Fecha diagnostico"] != '':
+                case["events"] = [
+                    {
+                        "name": "confirmed",
+                        "dateRange":
+                        {
+                            "start": convert_date(entry["Fecha diagnostico"]),
+                            "end": convert_date(entry["Fecha diagnostico"])
+                        }
+                    },
+                ]
+            elif entry["Fecha diagnostico"] == '':
```

Comment:
we'll need an `else:` after those if/elif and throw in case we can't get a confirmed date.

Reply:
I think I can just change this elif statement to else directly right? because there's only two scenarios: 
entry["Fecha diagnostico"] != '' or entry["Fecha diagnostico"] == ''

Reply:
if there are only two scenarios then yes else: is fine.